recipes/conversation_view/attachment_carousel/attachment_carousel.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+  <div
+    class="dt-attachment-carousel"
+    role="presentation"
+    @mouseenter="showCarouselArrows = true"
+    @mouseleave="showCarouselArrows = false"
+    @focusin="showCarouselArrows = true"
+    @focusout="showCarouselArrows = false"
+  >
+    <div
+      v-if="mediaList.length > 0"
+      ref="carousel"
+      class="dt-attachment-carousel--media-list"
+      :aria-label="attachmentAriaLabel"
+      @scroll="handleScroll"
+    >
+      <!-- Image list -->
+      <div
+        v-for="(mediaItem, index) in mediaList"
+        :key="`media-${index}`"
+        role="presentation"
+        class="d-ps-relative"
+        @focusin="closeButton(true, index)"
+        @focusout="closeButton(false, index)"
+        @mouseenter="closeButton(true, index)"
+        @mouseleave="closeButton(false, index)"
+      >
+        <dt-image-viewer
+          image-button-class="d-h64 d-w64 d-bar4 d-ba d-baw6 d-bc-subtle"
+          :image-src="mediaItem.path"
+          :image-alt="mediaItem.altText"
+          :close-aria-label="closeAriaLabel"
+          :aria-label="clickToOpenAriaLabel"
+        />
+        <dt-button
+          v-show="showCloseButton[index]"
+          class="dt-attachment-carousel--close-button"
+          circle
+          size="xs"
+          importance="clear"
+          :aria-label="closeAriaLabel"
+          @click="removeMediaItem(index)"
+        >
+          <template #icon>
+            <dt-icon
+              name="close"
+              size="200"
+            />
+          </template>
+        </dt-button>
+      </div>
+    </div>
+
+    <!-- Carousel Arrows -->
+    <dt-button
+      v-if="showLeftArrow && showCarouselArrows"
+      tabindex="-1"
+      class="dt-attachment-carousel--left-arrow d-ba d-baw1 d-bc-default"
```

Comment:
Remove utility classes and add them into the custom class (same goes for any others in this PR)